packages/workshop-app/app/components/diff.tsx

```diff
@@ -215,7 +215,7 @@ function SelectFileToDiff({
 				<Select.Content
 					position="popper"
 					align="start"
-					className="z-20 max-h-[70vh] overflow-scroll bg-black text-white"
+					className="z-20 max-h-[70vh] overflow-hidden bg-black text-white"
```

Comment:
```suggestion
					className="z-20 max-h-[70vh] bg-black text-white"
```

I double checked and this overflow class is redundant

Reply:
done